### PR TITLE
remove all nbdev-related files

### PR DESCRIPTION
**Pull request by jalew188**
1. Remove all nbdev-related files.
2. Use `python -m pytest --nbmake` instead of `nbdev_test` for all nbdev_nbs.

### CODE REVIEW

File: extra_requirements/development.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ twine
 bumpversion
 pipdeptree
 ipykernel
-nbdev
+# nbdev
```
Comment (mschwoer) [inline extra_requirements/development.txt:6]: just delete it :-)

File: tests/run_tests.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+INCLUDED_NBS=$(find ./nbdev_nbs -name "*.ipynb")
+python -m pytest --nbmake $(echo $INCLUDED_NBS)
+
+
+INCLUDED_NBS=$(find ./docs/nbs -name "*.ipynb" | grep -v tutorial_dev_spectral_libraries.ipynb)
```
Comment (mschwoer) [inline tests/run_tests.sh:5]: did you drop the `TODO` comments on purpose? I thing they served well explaining why we exclude some notebooks here ..

File: .github/workflows/_run_tests.yml [hunk 3]
```diff
@@ -37,27 +37,9 @@ jobs:
...
-    - name: Test additional notebooks
-      shell: bash -l {0}
-      # TODO make test_isotope_mp.ipynb work
-      # Note: multiprocessing in ipynb sometimes suspended on some versions of Windows, ignore the
-      # corresponding notebook(s) if this occurs again
-      run: |
-        conda activate alphabase
-        INCLUDED_NBS=$(find ./nbs_tests -name "*.ipynb" | grep -v test_isotope_mp.ipynb)
-        python -m pytest --nbmake $(echo $INCLUDED_NBS)
+        . ./tests/run_tests.sh
```
Comment (mschwoer) [inline .github/workflows/_run_tests.yml:44]: +1
good idea moving these to a script!

**Review by mschwoer — APPROVED** [review]
LGTM!

File: tests/run_tests.sh (hunk 2 above)
Comment (jalew188) [reply tests/run_tests.sh:5]: Yes, I will move them back

**Pull request merged**